dune/xt/la/container.hh

```diff
@@ -12,6 +12,8 @@
 #ifndef DUNE_XT_LA_CONTAINER_HH
 #define DUNE_XT_LA_CONTAINER_HH
 
+#include <boost/tuple/tuple.hpp>
```

Comment:
What about ``std::tuple``?

Reply:
I am using these in the python bindings (copied code from @renefritze to automagically add all types within the tuple), and I think it has to be the boost tuple for that...

Reply:
Okay, I was just wondering because I thought ``boost::tuple`` is essentially the same as ``std::tuple`` as long as you are using ``C++11``.